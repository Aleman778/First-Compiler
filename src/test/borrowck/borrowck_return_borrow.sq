// ignore

fn main() {
    let a: &i32 = returning_reference();
    print_int(*a);
}


fn returning_reference() -> &i32 {
    let a: &i32 = &42;
    a    
}
