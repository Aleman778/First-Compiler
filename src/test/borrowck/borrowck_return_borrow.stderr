error: missing lifetime specifier
  --> $DIR/borrowck_return_borrow.sq:10:29
   |
LL | fn returning_reference() -> &i32 {
   |                             ^ help: consider giving it a 'static lifetime: `&'static`
   |